thunderbird/default.cfg

```diff
@@ -10,6 +10,7 @@ provider_url=https://thunderbird.readthedocs.org/en/latest/
 url = http://localhost:5001/wps
 outputurl = http://localhost:5001/outputs
 allowedinputpaths = /
+maxrequestsize = 200mb
```

Comment:
Why this size?

Reply:
I have no particular reason. I just matched it with the `maxsingleinputsize`. And we do not have local test data that is very large as well.